### PR TITLE
Renames

### PR DESCRIPTION
**Pull request by steveoh**
I moved the models into models.py but didn't update the tests. I started to mock out the dirty stuff as well. You can run with this hopefully.

### CODE REVIEW

File: src/forklift/models.py [hunk 1]
```diff
@@ -53,3 +57,20 @@ def add_crates(self, crate_infos, defaults={}):
...
+        #: the name of the source data table
+        self.source_name = source_name
+        #: the name of the source database
+        self.source = source
+        #: the name of the destination database
+        self.destination = destination
+        #: the name of the output data table
+        self.destination_name = destination_name or source_name
+        #: the unique name of the crate
+        self.name = '{}::{}'.format(source, source_name)
```
Comment (stdavis) [inline src/forklift/models.py]: I think that this should be destination, shouldn't it? There are instances where the same source goes to two different destinations. We only want to skip if two crates are pointed at the same destination.

Comment (steveoh) [inline src/forklift/models.py]: yeah that would be safer probably for uniqueness

**Comment by stdavis** [comment]
I think that we may be ready for another merge. All tests are passing. Coverage is pretty good. The next step as I see it is the logging/reporting.

**Comment by steveoh** [comment]
This is exciting

**Pull request closed without merging**